app/src/main/java/org/listenbrainz/android/ui/screens/yim23/Yim23LandingScreen.kt

```diff
@@ -103,10 +104,10 @@ fun Yim23HomeScreen(
                                     Image(
                                         painter =
                                         painterResource(id = when (viewModel.themeType.value) {
-                                            0 -> R.drawable.yim23_pick_color_green
```

Comment:
In the `Yim23ThemeData` (Renamed to Yim23ThemeType) enum class, create a parameter with name `pickColorRes` and use `viewModel.themeType.pickColorRes` everywhere instead of running a when clause every time.
add other fields as well like `colorRes`, `logoRes` etc.